stdlib/public/SDK/XCTest/XCTest.swift

```diff
@@ -1030,6 +1030,40 @@ public func XCTAssertThrowsError<T>(_ expression: @autoclosure () throws -> T, _
   }
 }
 
+public func XCTAssertNoThrow<T>(_ expression: @autoclosure () throws -> T, _ message: @autoclosure () -> String = "", file: StaticString = #file, line: UInt = #line, _ errorHandler: (_ error: Error) -> Void = { _ in }) {
```

Comment:
I guess it's a good thing because you only fixed one level of indentation!  If you're using Xcode, set your indentation settings appropriately and try highlighting the regions I mentioned.  Hit `^+I` and it'll reindent for you.